### ISSUE TITLE
Job logs should warn instead of error

### ISSUE THREAD

**Issue by jsafarik** [jsafarik/multi-ocp-monitoring#3]
At this moment, when something can't be fetched from the cluster (for example accessibility job can't fetch the console URL), error log is always used. Instead, warn log should be used on most places, as error log should be used for issues that goes wrong with the monitoring application itself.
